doc/nrf/releases_and_maturity/releases/release-notes-2.9.2.rst

```diff
@@ -0,0 +1,125 @@
+.. _ncs_release_notes_292:
+
+|NCS| v2.9.2 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF54, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.9.0 <ncs_release_notes_290>` and :ref:`nRF Connect SDK v2.9.1 <ncs_release_notes_291>`.
+
+.. todo
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.9.2**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.9.2`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.9.2.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.9.2_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.9.2_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.9.2`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_292_changelog:
+
+Changelog
+*********
+
```

Comment:
@divipillai 
I would like to add following changelog entry for the following sdk-zephyr PR: https://github.com/nrfconnect/sdk-zephyr/pull/2836
```
Drivers
=====

nrfx
----

* Changed:

  * Updated MDK to version 8.68.2 to have improved errata handling for nRF54L05 and nRF54L10 devices.
```

is `Drivers` section correct here? this is `sdk-zephyr`-related rather than `sdk-nrf`-related

Reply:
We normally have a dedicated section for nrfx.  For example - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.5.1.html#nrfx. 
Added the entry.